pkg/scheduler/scheduler.go

```diff
@@ -188,21 +190,62 @@ func (s *Scheduler) nominate(ctx context.Context, workloads []workload.Info, sna
 	return entries
 }
 
+type admissionStatus struct {
+	podSet       string
+	resourceName string
+	reasons      []string
+	err          error
+}
+
+// Message returns a concatenated message on reasons of the admissionStatus.
+func (s *admissionStatus) Message() string {
+	if s == nil {
+		return ""
+	}
+	msg := strings.Join(s.reasons, ", ")
+	return fmt.Sprintf("Workload didn't fit, insufficient %s in %s, reasons: %s;", s.resourceName, s.podSet, msg)
```

Comment:
if it's an error, it's not about insufficient quota

Reply:
Then I'll change to report error from `admissionStatus.err`, and report reasons from `admissionStatus.reasons`.